src/cookies.zig

```diff
@@ -89,48 +90,67 @@ test Cookie {
 
 pub const Jar = struct {
     alloc: std.mem.Allocator,
-    cookies: []Cookie,
+    cookies: ArrayListUnmanaged(Cookie),
     capacity: usize = 0,
 
+    /// Creates a new jar.
     pub fn init(a: std.mem.Allocator) !Jar {
-        var cookies = try a.alloc(Cookie, 8); // 8 aught to be enough for anyone! :D
-        cookies.len = 0;
         return .{
             .alloc = a,
-            .cookies = cookies,
+            .cookies = ArrayListUnmanaged(Cookie){},
             .capacity = 8,
```

Comment:
lets delete capacity if we're gonna use array list

Reply:
Done